scripts/completions/harnx.bash

```
@@ -58,6 +53,15 @@ _harnx() {
                     __ltrim_colon_completions "$cur"
                     return 0
                     ;;
+                --serve)
+                    COMPREPLY=($(compgen -f "${cur}"))
```

Shellcheck (0.11.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)